### DISCUSSION TITLE
YouTube HLS Server

### DISCUSSION THREAD

**Discussion by alerickert** [abskmj/notes#24] [General]
Hello, I bought you: Youtube HLS Server
but I can't configure it, and what I need is my own url to do this:

https://ythls-v2.onrender.com/channel/$youtube_channel_id.m3u8

I place the video streams in my clients' apps, now I am using your url but you say that if there are many requests it fails.

I use a json api, and each app reviews the data that is in that api.

Anyway, I need to not have blockage problems, I sent you an email to see if we can chat. Thank you

**Comment by alerickert** [comment]
I already managed to make it work in my url.

root@debian-2gb-ash-3:~# PORT=80 ./youtube-hls-server
server listening on port: 80
request url: /
request url: /favicon.ico
request url: /
request url: /favicon.ico
request url: /channel/UCgn6r0aGRBnEQm02tE_jzbw.m3u8
request for channel with id: UCgn6r0aGRBnEQm02tE_jzbw
request url: /channel/UCgn6r0aGRBnEQm02tE_jzbw.m3u8
request for channel with id: UCgn6r0aGRBnEQm02tE_jzbw
request url: /channel/UCgn6r0aGRBnEQm02tE_jzbw.m3u8
request for channel with id: UCgn6r0aGRBnEQm02tE_jzbw
request url: /channel/UCgn6r0aGRBnEQm02tE_jzbw.m3u8
request for channel with id: UCgn6r0aGRBnEQm02tE_jzbw



1) How to make thousands of requests without problems?
Is there a way for the query to be made and saved for 60 minutes and the next people who open the url do so from a saved file? I have stations with 5000 connections, that's why I put the url in the app, but if a query is made every day or every so often, are they going to block me or not?

2) Does this process consume large ram/cpu? I put it on the smallest vps in the data center I use: 2 cpu 2 ram.

3) I have the url in cloudflare with a proxy, does that help?

4) do you have this same thing for twitch?

**Comment by abskmj** [comment]
We discussed these over the email.
